org.flameshot.Flameshot.yml

```diff
@@ -14,11 +14,8 @@ finish-args:
   - --share=network
   # QtSingleApplication, allow other instances to see log files
   - --env=TMPDIR=/var/tmp
-  # Notification access
-  - --talk-name=org.freedesktop.Notifications
```

Comment:
Is notifications not required?

Reply:
I tried to build it here to test if it works. In freedesktop 23.08 it is not needed so I thought here it shouldn't be too. But looks like it's not showing notifications. Maybe flameshot has to do something on their side?

Reply:
> In freedesktop 23.08 it is not needed

It's not needed if it uses libnotify and I don't think flameshot does.

libnotify is also not part of Freedesktop SDK 23.08, you are confusing it with the Electron baseapp 23.08 which includes it

Reply:
Thanks for the clarification. I really appreciate it. :)